tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestExampleGenerator.java

```diff
@@ -1645,4 +1647,81 @@ public void TestUnicodeSetExamples() {
             assertEquals(locale + path + "=" + value, expected, actual);
         }
     }
+
+    public void TestEraMap() {
+        ExampleGenerator exampleGenerator = getExampleGenerator("en");
+        Relation<String, String> keyToSubtypes = SupplementalDataInfo.getInstance().getBcp47Keys();
+        Set<String> calendars = keyToSubtypes.get("ca"); // gets calendar codes
+        Map<String, String> codeToType = new HashMap<String, String>() {{ // calendars where code != type
+            put("gregory", "gregorian");
+            put("ethioaa", "ethiopic-amete-alem");
+            put("islamic-civil", "islamic");
+            put("islamic-rgsa", "islamic");
+            put("islamic-tbla", "islamic");
+            put("islamic-umalqura", "islamic");
+            put("islamicc", "islamic");
+        }};
+        for (String id : calendars) {
+            if (id.equals("iso8601")) { // generic
```

Comment:
iso8601 should be behave like gregorian, so you should go ahead as map it above and remove this.

Reply:
Got it